client/src/actions/ActionTypes.js

```diff
@@ -0,0 +1,10 @@
+import keyMirror from 'keymirror';
+
+export default keyMirror({
+  AUTH_ERROR: null,
+  CHECK_IN_SUCCESS: null,
+  SIGN_OUT: null,
+
```

Comment:
also good practice is to use naming patterns like `DID_SIGN_OUT` and `GOT_AUTH_ERROR`